dirty_cat/gap_encoder.py

```diff
@@ -500,20 +398,14 @@ def transform(self, X):
 
         Parameters
         ----------
-        X : array-like, shape (n_samples, ) or (n_samples, 1)
+        X : array-like, shape (n_samples, )
```

Comment:
Isn't this now (n_samples, n_features)?
```suggestion
        X : array-like, shape (n_samples, n_features)
```
(should be propagated everywhere)

Reply:
Here it should be just `(n_samples, )` because we are in the `GapEncoderColoumn` class. The GapEncoder expects 2D arrays, but the GapEncoderColumn expect 1D arrays.